src/test/java/org/fundacionjala/coding/escarleth/Kata4Test.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.escarleth;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * This class contains UnitTests for Kata4.
+ */
+public class Kata4Test {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)